lib/transport/tests/test_aux_data.c

```diff
@@ -20,7 +20,8 @@
  * OpenSSL libraries as published by the OpenSSL project. See the file
  * COPYING for details.
  */
-#include "testutils.h"
+#include <criterion/criterion.h>
+#include "apphook.h"
 #include "transport/transport-aux-data.h"
 
 #define AUX_DATA_TESTCASE(x, ...) do { aux_data_testcase_begin(#x, #__VA_ARGS__); x(__VA_ARGS__); aux_data_testcase_end(); } while(0)
```

Comment:
Could you please remove this unused macros ?

Reply:
Ok. I will remove those macros.

Reply:
ok